test/unittests/commands/xmlupload/stash/test_stash_circular_references_from_intermediary_resource.py

```diff
@@ -0,0 +1,187 @@
+from copy import deepcopy
+from uuid import uuid4
+
+import pytest
+
+from dsp_tools.commands.xmlupload.models.formatted_text_value import FormattedTextValue
+from dsp_tools.commands.xmlupload.models.intermediary.res import IntermediaryResource
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryLink
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediaryRichtext
+from dsp_tools.commands.xmlupload.models.intermediary.values import IntermediarySimpleText
+from dsp_tools.commands.xmlupload.models.permission import Permissions
+from dsp_tools.commands.xmlupload.models.permission import PermissionValue
+from dsp_tools.commands.xmlupload.stash.stash_circular_references_from_intermediary_resource import (
+    stash_circular_references,
+)
+from dsp_tools.commands.xmlupload.stash.stash_models import LinkValueStash
+from dsp_tools.commands.xmlupload.stash.stash_models import StandoffStash
+from dsp_tools.commands.xmlupload.stash.stash_models import Stash
+
+
+@pytest.fixture
+def simple_text_value() -> IntermediarySimpleText:
+    return IntermediarySimpleText("Text", "prop", None, None)
```

Comment:
One thought concerning these fixtures: 

- With 1 exception, they are only used once. 
- Most of them are 1-liners
- That means, most of them could be inlined.

Inlining them would greatly improve the [Locality of Behaviour](https://htmx.org/essays/locality-of-behaviour/). As the linked essay states, LoB might be in conflict with other principles. So it's up to the developers to find a trade-off between conflicting principles.

In this concrete case, I would find it easier if no fixtures are used. But I leave that decision up to you :-)